hypothesis-python/tests/nocover/test_floating.py

```diff
@@ -110,7 +114,7 @@ def test_is_in_exact_int_range(x):
 # compiler options like `-ffast-math`.  On most hardware that's even a global
 # config option, so *linking against* something built this way can break us.
 # Everything is terrible
-FLUSH_SUBNORMALS_TO_ZERO = next_down(sys.float_info.min) == 0.0
+FLUSH_SUBNORMALS_TO_ZERO = next_down(float_info.min) == 0.0
```

Comment:
Let's make this a new test module, `nocover/test_subnormal_floats.py`, so that we can have our `skipif` as a module-global mark.  Includes moving these two existing tests.

Reply:
I ended up rewriting the existing tests, I think it's more clear now. I _think_ the given/fails thing is not necessary after #3152 .